DrewBradley/rancid-tomatillos
# Test/testing header component

### What does this PR do?
- This Pr tests our header component contains the proper elements and it's functionality.

### What does it fix?
- Nothing to fix.

### Is this a feature or a fix?
- This is testing, confirming our header has the proper elements and that it filters movies by title based on user input.

### Where should the reviewer start?
- Clone this repo down to your machine. Run npm install then run npm start. 

### How should this be tested?
- We've already run the necessary tests and the acceptance criteria is met. You can install cypress, open cypress, and verify that all tests are passing. Make sure to remove the "only" statement from any it blocks.
